packages/protocol/contracts/identity/Escrow.sol

```diff
@@ -115,37 +172,61 @@ contract Escrow is
     address paymentId,
     uint256 minAttestations
   ) external nonReentrant returns (bool) {
-    require(token != address(0) && value > 0 && expirySeconds > 0, "Invalid transfer inputs.");
-    require(
-      !(identifier.length <= 0 && !(minAttestations == 0)),
-      "Invalid privacy inputs: Can't require attestations if no identifier"
-    );
-
-    IAttestations attestations = IAttestations(registry.getAddressFor(ATTESTATIONS_REGISTRY_ID));
-    require(
-      minAttestations <= attestations.getMaxAttestations(),
-      "minAttestations larger than limit"
-    );
-
-    uint256 sentIndex = sentPaymentIds[msg.sender].push(paymentId).sub(1);
-    uint256 receivedIndex = receivedPaymentIds[identifier].push(paymentId).sub(1);
-
-    EscrowedPayment storage newPayment = escrowedPayments[paymentId];
-    require(newPayment.timestamp == 0, "paymentId already used");
-    newPayment.recipientIdentifier = identifier;
-    newPayment.sender = msg.sender;
-    newPayment.token = token;
-    newPayment.value = value;
-    newPayment.sentIndex = sentIndex;
-    newPayment.receivedIndex = receivedIndex;
-    // solhint-disable-next-line not-rely-on-time
-    newPayment.timestamp = now;
-    newPayment.expirySeconds = expirySeconds;
-    newPayment.minAttestations = minAttestations;
+    address[] memory trustedIssuers;
+    // If minAttestations == 0, trustedIssuers should remain empty
+    if (minAttestations > 0) {
+      trustedIssuers = getDefaultTrustedIssuers();
```

Comment:
in theory in the future we may have a more complicated way of getting default trusted issuers (via a separate contract, possibly a token-curated registry) but this may be overkill for now